tests/torch/test_compile_torch.py

```diff
@@ -1144,7 +1147,12 @@ def test_shape_operations_net(
     check_float_array_equal,
 ):
     """Test a pattern of reshaping, concatenation, chunk extraction."""
-    net = ShapeOperationsNet(is_qat)
+    net = model(is_qat)
+    if is_qat and not net.is_qat_compatible:
+        pytest.skip(f"Model {model.__name__} is not compatible with is_qat=True")
+
+    # Shape transformation do not support >1 example in the inputset
+    # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/3871
     inputset = numpy.random.uniform(size=(1, n_channels, 2, 2))
 
     if is_qat:
```

Comment:
just realized by reading what's following that in case of `is_qat=False`, we never check any r2 score or whatever. Basically, this PR is green only because the forward/compile didn't fail ! There is only this comment bu no FIXME attached to it : is this still expected ? : 
```
    # In QAT quantization options are consistent across all the layers
    # which allows for the elimination of TLUs
    # In PTQ there are TLUs in the graph because Shape/Concat/Transpose
    # must quantize inputs with some default quantization options

    # In QAT testing in FHE is fast since there are no TLUs
    # For PTQ we only test that the model can be compiled and that it can be executed
```

Reply:
Well with 3 bits we can't expect the values to be the same. However, we are only soing shape transformation so I guess that only the output shape should be checked. Not sure why we check the actual prediction at all.

Reply:
I see, well could you maybe improve that ? because I believe that when `is_qat` is False (which is your case with your added model), we do not check the shape !